src/chess/Chess.java

```diff
@@ -1,5 +1,5 @@
 
-package Chess;
+package chess;
```

Comment:
If the package name is chess, should the file be chess.java instead of Chess.java? (I'm not well versed in Java conventions either)